zerver/lib/bugdown/__init__.py

```diff
@@ -568,6 +613,20 @@ def vimeo_image(self, url: Text) -> Optional[Text]:
             return "http://i.vimeocdn.com/video/%s.jpg" % (vm_id,)
         return None
 
+    def message_quote_id(self, url: Text) -> Optional[Text]:
+        url_split = url.split('/')
+
+        if not (url_split[url_split.__len__() - 2] == 'near'):
```

Comment:
I might be wrong, but this funcion vould have false positives if we paste any Zulip links from another realm, or even coincidentally similar links to what our url format is. Perhaps a check for the host of the url should be helpful here.